src/main/webapp/style.css

```diff
@@ -1,5 +1,6 @@
 body {
   color: black;
+  font-family: Roboto,Arial,sans-serif;
```

Comment:
is it possible to make everything just roboto?

Reply:
This means that if Roboto can't be used for whatever reason, use Arial and then sans-serif as backups. So, if the browser is able to render Roboto, then everything is Roboto